src/backends/torch/torchlib.cc

```diff
@@ -965,8 +974,9 @@ namespace dd
         Tensor output;
         try
           {
-            if (extract_layer.empty())
-              output = torch_utils::to_tensor_safe(_module.forward(in_vals));
+            if (extract_layer.empty() || extract_last)
```

Comment:
so extract_last does exactly the same as extract_layer.empty() ?

Reply:
No, `extract_layer` set to `"last"` bypasses the call to `extractable()`. This is a bit hacky, but I did it for two reasons:
- `extractable()` is not implemented on our side for traced models yet
- it unlocks the useful and generic use-case of running a network an returning the flatten tensors from the last layer. GAN generator and even segmentation models fall within this category. Though some elements should be refined in the future (e.g. non-flatten tensors).

Reply:
Ok, I missed the fact that if extract_layer != "" we already return a tensor